filebeat/input/filestream/environment_test.go

```diff
@@ -197,15 +212,15 @@ func (e *inputTestingEnvironment) requireOffsetInRegistry(filename string, expec
 }
 
 // requireMetaInRegistry checks if the expected metadata is saved to the registry.
-func (e *inputTestingEnvironment) waitUntilMetaInRegistry(filename string, expectedMeta fileMeta) {
+func (e *inputTestingEnvironment) waitUntilMetaInRegistry(filename, inputID string, expectedMeta fileMeta) {
```

Comment:
The header comment seems incorrect here. But it's not new, so you can ignore this problem.

Reply:
indeed. Well CI is red anyways, so I might as well fix this ;)